kubecost/templates/_helpers.tpl

```diff
@@ -548,21 +548,24 @@ federated storage config helpers
   {{- end }}
 {{- end -}}
 
+    {{/*
+    NOTE: added kubecostModel for backward compatibility
+    */}}
 {{- define "kubecost.federatedStorage.config" }}
   {{- if (.Values.kubecostModel).federatedStorageConfig -}}
-    {{- (.Values.kubecostModel).federatedStorageConfig -}}
-  {{- else if (.Values.federatedStorage).config }}
-    {{- (.Values.federatedStorage).config -}}
+    {{- toYaml (.Values.kubecostModel).federatedStorageConfig -}}
```

Comment:
Are you sure here? In your PR description you say you test by doing the following:

```yaml
kubecostModel:
  federatedStorageConfig:
    type: S3
    config:
      bucket: xxx
```

But I thought that historically this storage config was passed as a string, not an object? https://github.com/kubecost/kubecost/blob/33a39985a5c6641d339a29f177440efaaf2b795c/cost-analyzer/values.yaml#L633-L670

Reply:
you are right! 
okay, fixed everything I think.